Readme.md

```diff
@@ -4,13 +4,15 @@
 [![npm][npm-badge]][npm]
 [![Discord][discord-badge]][discord]
 
-This repo provide tools to export Nightwatch test from Google Chrome Devtools Recordings
+This repo provide tools to convert JSON user flows from [Google Chrome DevTools Recorder](https://goo.gle/devtools-recorder) to Nightwatch test scripts programmatically.
 
 ✅ Converts multiple recordings to Nightwatch tests in one go (out-of-the-box glob support)  
 🗂 User can pass their custom path to export tests.  
 💃 Users can also use a dry run to see the interim output of the recordings  
 👨‍💻 Programmatic API which users can use in their own project to create plugins or custom scripts.
 
+Alternatively, you can export JSON user flows as Nightwatch test scripts straight away from Chrome DevTools with our [Nightwatch Recorder Chrome extension](https://github.com/nightwatchjs/nightwatch-recorder-extension).
```

Comment:
Should we mention the Chrome Webstore link instead?

Reply:
Fixed